__tests__/index.tsx

```diff
@@ -1,14 +1,60 @@
 import * as React from 'react';
-import {shallow} from 'enzyme';
+import { shallow, render, mount } from 'enzyme';
+import Circle from '../src';
 
 describe('ReactCircle', () => {
   const setup = () => {
     return {
       
     };
   };
-  
-  it('Foo', () => {
-    expect(1).toEqual(1);
-  });
+
+  const initialOptions = {
+    progress: 25,
+    progressColor: '#F7DC1B',
+    bgColor: '#54BAD8',
+    textColor: 'hotpink',
+    size: '200',
+    lineWidth: `30`,
+    showPercentage: true
+  }
+
+  it('Should render a <Circle/> with text percentage', () => {
+    const CircleNode = shallow(
+      <Circle {...initialOptions}/>
+    )
+    expect(CircleNode).toMatchSnapshot();
```

Comment:
I don't think having the whole tree as snapshot add much value, I'd rather prefer to be more detailed and cover specific cases, thoughts?